src/ui/components/Library/Team/View/NewTestRuns/TestRuns.module.css

```diff
@@ -42,7 +42,7 @@
 }
 
 .failedPill {
-  color: var(--theme-base-100);
+  color: var(--primary-accent-foreground-text);
   background-color: var(--testsuites-failed-color);
```

Comment:
This looks wrong to me. `--testsuites-failed-color` is meant to be a _foreground_ color, but this is using it as a _background_ color and stacking another arbitrary color on top of it.

Do you mean to be using `--testsuites-capsule-error-bgcolor` and `--testsuites-capsule-error-color` here instead?

Reply:
@jonbell-lot23 Can you please help here? I think you recently did a pass on improving the background color here. I'm not sure which color do we need to use.

For text color, I want to use white on top of testsuite's red. I'm also not sure which variable to use here.

Reply:
> For text color, I want to use white on top of testsuite's red. I'm also not sure which variable to use here.

`--testsuites-capsule-error-bgcolor` and `--testsuites-error-icon-bgcolor` are both solid red, where as `--testsuites-error-color` is pinkish. Not sure why that is. I dunno what you should be using then, but I don't think you should be using what's currently used above.

Reply:
Sorry @callingmedic911 , I just saw this. Looking now.

Reply:
1. The reason --testsuites-error-color is pinkish in dark theme is because it's set on a dark maroon background, and red on near-black is hard to read: 
<img width="306" alt="image" src="https://github.com/replayio/devtools/assets/9154902/c48dae42-a3b4-4080-885e-a10148f07fd3">

2. @callingmedic911 , you'll want to use --testsuites-capsule-error-color for the color, and --testsuites-capsule-error-bgcolor for the background. See how this class is called "pill?" The aligning naming is "capsule" in the CSS.

Reply:
I think similar changes would be required for flakyPill and successPill too - for background colors? Can I request a separate PR to fix the colors too, after we merge this one? I'm copying these classes over from TestRuns.module.css to TestListItemModule.css, I'm going to revert that back to these original values in this PR. To be clear, we'd need to update both of these module in follow up PR.

Reply:
Filed a task here: https://linear.app/replay/issue/DES-1027/follow-up-color-pr